src/Tribe/Commerce/Warnings.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Conditional Warnings on Ticket Features
+ *
+ * Class Tribe__Tickets__Commerce__Warnings
+ *
+ * @since TBD
+ *
+ */
+class Tribe__Tickets__Commerce__Warnings {
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 4d56c4e000d83295eabee10dd25ef73b</sub>